src/cpp/session/resources/schema/user-prefs-schema.json

```diff
@@ -321,8 +323,9 @@
         "surround_selection": {
             "type": "string",
             "enum": ["never", "quotes", "quotes_and_brackets"],
+            "enumReadable": ["Never", "Quotes", "Quotes & Brackets"],
             "default": "quotes_and_brackets",
-            "title": "Surround selections with",
+            "title": "Surround selections with delimiter insertion",
```

Comment:
Though this seems like a reasonable correction, let's not mess with this unless absolutely necessary for this PR.